exercises/practice/sum-of-multiples/tests/sum_of_multiples.cairo

```diff
@@ -0,0 +1,176 @@
+use sum_of_multiples::{sum_of_multiples as verify_sum_of_multiples};
+
+#[test]
+fn no_multiples_within_limit() {
+    let mut factors: Array<u32> = ArrayTrait::new();
+    factors.append(3);
+    factors.append(5);
+    let limit = 1;
+    let output = verify_sum_of_multiples(limit, factors);
+    assert(output == 0, 'Test failed: Expected 0.');
+}
+
+#[test]
+#[ignore]
+fn one_factor_has_multiples_within_limit() {
+    let mut factors: Array<u32> = ArrayTrait::new();
```

Comment:
We could use the `array!` macro from https://book.cairo-lang.org/ch03-01-arrays.html#array-macro to declare the factors array and build it as a single line. That should make the test suite more concise and easier for students to read.